### ISSUE TITLE
Add "admin privilege" chip to dandiset listings

### ISSUE THREAD

**Issue by waxlamp** [dandi/dandi-archive#838]
In addition to the warnings shown to admins inside of DLPs for dandisets they don't own, we should show this status in the listings themselves. This will be important for embargoed dandisets, especially if we only show those in the "my dandisets" listing as suggested in #834.

@mvandenburgh, thoughts?

**Comment by mvandenburgh** [comment]
Agreed that we need some kind of indicator. I wonder if it would be better to have an icon with a tooltip that gives this info, to differentiate it from the other chips as an "admin-only" feature. It's such a small thing that I could implement both and we can decide which looks better.

**Comment by waxlamp** [comment]
A red-background `#`, with an explanatory tooltip, could be a good choice (reflecting the standard use of `#` for sudo shells in Linux). To avoid showing this status on all public dandisets not owned by an admin, I think this feature would be specific to the "my dandisets" view (to mark them as coming under admin purview even though not owned by the admin).
